dev/criteo/test-with-maven.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# You can run tests for a module only by specifying -pl <module>
+# as argument to this script, or some specific test by passing
+# -Dtest=... (see surefire documentation).
+
+source `dirname "$0"`/criteo-spark-env.sh
+
+export _JAVA_OPTIONS="-Xss2048k -Dspark.buffer.pageSize=1048576 -Xmx4g"
```

Comment:
I do not know, I just replayt he setting one spark developper published on the sparl-dev mailing list. Feel free to adjust - or I'll do it next push